### PR TITLE
LOG-6898 Removed fake X509TrustManager code; Logentries client does n…

### PR DESCRIPTION
**Pull request by pclune-r7**
…ot use a custom TrustManager to override default TrustManager

### CODE REVIEW

**Comment by pclune-r7** [comment]
@cheffernan-r7 @mrud

File: src/main/java/com/logentries/net/LogentriesClient.java [hunk 1]
```diff
@@ -227,23 +227,8 @@ private SSLContext CreateAndInitSSLContext(String algorithm) throws NoSuchAlgori
 			KeyManagementException {
 
 		SSLContext sslContext = SSLContext.getInstance(algorithm);
```
Comment (udangel-r7) [inline src/main/java/com/logentries/net/LogentriesClient.java]: shouldn't we use the `getDefault()` message

Comment (pclune-r7) [inline src/main/java/com/logentries/net/LogentriesClient.java]: we could - but as you can see the code is written to allow creation of an instance of the context with a choice of protocol - why? I have no idea, but I'm sure it seemed like a good idea to however wrote it. I am deliberately trying to minimise changes to this library - would it be a big deal to leave it as is?

http://developer.android.com/reference/javax/net/ssl/SSLContext.html#getInstance%28java.lang.String%29

Comment (mrud) [inline src/main/java/com/logentries/net/LogentriesClient.java]: I think it does not make much sense to use a custom context if we do not use it. 
The idea would be to get the SSL behaviour correct

Comment (pclune-r7) [inline src/main/java/com/logentries/net/LogentriesClient.java]: hhhmmmm...yes, I see what you mean - and I see a couple of other issue - oh well - back to the drawing board.

**Pull request closed without merging**